src/web/js/ide.js

```diff
@@ -10,11 +8,45 @@ function loadScriptUrl(url) {
   document.body.appendChild(scriptTag);
 }
 
-// this is needed by pyret I guess.
-require('script!requirejs/require.js');
-window.define('seedrandom', [], function() { return seedrandom; });
-loadScriptUrl(process.env.BASE_URL+'/js/cpo-ide-hooks.jarr');
 
 var appDiv = document.createElement("div");
 document.body.appendChild(appDiv);
-ReactDOM.render(React.createElement(PyretIDE, null), appDiv);
+
+const runtimeApi = {
+  parse(src) {
+    return new Promise((resolve, reject) => reject(new Error("Not Implemented")));
+  },
+  compile(ast) {
+    return new Promise((resolve, reject) => reject(new Error("Not Implemented")));
+  },
+  execute(bytecode) {
+    return new Promise((resolve, reject) => reject(new Error("Not Implemented")));
+  },
```

Comment:
I just need some help figuring out how to fill these in using the APIs available on window.CPOIDEHooks (which are defined in cpo-ide-hooks.js)